draft-wang-ppm-dap-taskprov.md

```diff
@@ -296,6 +296,23 @@ A protocol participant MUST opt out if the task has expired.
 The behavior of each protocol participant is determined by whether or not they
 opt in to a task.
 
+## Supporting a Global HPKE Configuration {#global-hpke-config}
+
```

Comment:
editorial: I suggest reworking this so that all of the motivation is in the first paagragh, then all normative text follows:

> Clients need to know the HPKE configuration prior to uploading reports, but 
>
> To mitigate this issue, Aggregators SHOULD ...
>
> In addition, Clients SHOULD NOT ...

Reply:
Good point, there are three paragraphs: 1. describe the issue of specifying `task_id` in `/hpke_config`, 2. Aggregator's recommended behavior, 3. Client's recommended behavior.